include/vcpkg/base/system.process.h

```diff
@@ -45,6 +47,8 @@ namespace vcpkg
 
         Command& forwarded_args(View<std::string> args) &
         {
+            buf.reserve(buf.size() + args.size());
```

Comment:
This is not a correct reserve and this will almost always incur an additional allocation. (Since any escapes or adding of quotes will take more space)